docs/java/features/odata/use-vdm.md

```diff
@@ -466,3 +603,36 @@ POST /ODataService/API_BUSINESS_PARTNER/A_BusinessPartner(123)/to_BusinessPartne
 
 </TabItem>
 </Tabs>
+
+## Migrate to improved OData VDM (Beta)
```

Comment:
I would actually shorten the whole section down to reflect the real effort most consumers will have to put in which is very minor.

Reply:
If I was a consumer and would read this I would immediately think it's too much effort, just by looking at the size of the description compared to the payoff (beta functionality).

Reply:
Fine, but what would you omit? I mean, what is so irrelevant that we could leave it out?